ImageServer.py

```diff
@@ -0,0 +1,54 @@
+from URLtoPebblePng import get_pebble_png
+import SimpleHTTPServer
+import SocketServer
+import cgi
+import sys
+import urlparse
+
+PORT = 8000
+
+class ServerHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+
+    def do_GET(self):
+    	# For a get request, just return the Pebble_image.png
+    	# This is taken care of by the simeple HTTP server
+    	# request handler
+        return SimpleHTTPServer.SimpleHTTPRequestHandler.do_GET(self)
+
+    def do_POST(self):
```

Comment:
POST is not the right type of request for this. We are not changing the state on the server so we should be using a GET.